lib/isLatLong.js

```diff
@@ -16,6 +16,7 @@ function _default(str) {
   (0, _assertString.default)(str);
   if (!str.includes(',')) return false;
   var pair = str.split(',');
+  if (pair[0].charAt(0) === '(' && pair[1].charAt(pair[1].length - 1) !== ')' || pair[1].charAt(pair[1].length - 1) === ')' && pair[0].charAt(0) !== '(') return false;
```

Comment:
Can re-format/write this for better readability?
```javascript
if (pair[0].startsWith('(') && !pair[1].endsWith(')')
  || pair[1].endsWith(')') && !pair[0].startsWith('(') return false;
```

Reply:
I will work on that @profnandaa. 

Thank you.